include/singa/core/device.h

```diff
@@ -134,6 +150,8 @@ class Device {
   int num_executors_ = 0;
   unsigned seed_ = 0;
   bool graph_enabled_ = false;
+  string profiling_mode_ = "CHRONO";
+  int verbosity_ = 0;
```

Comment:
How about using two enum classes to distinguish the different profiling mode and the verbosity level?